compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -62,6 +62,21 @@ std::vector<uint8_t> genRandomBoolData(std::mt19937 &gen, uint32_t num_elements)
   return input_data;
 }
 
+template <typename T> std::vector<T> genRandomIntData(std::mt19937 &gen, uint32_t num_elements)
+{
+  std::uniform_int_distribution<T> dist(std::numeric_limits<T>::min(),
+                                        std::numeric_limits<T>::max());
```

Comment:
This code can generate extremely large values. Is it intended?

I don't know how the integer values will be used (maybe index?), but I think it would be better to restrict the range of generated values.

Reply:
I thought about it, but I have not come up with the best way to do it (which range should be?), and for the OneHot operation, the input data (indices) can be any. So, I decided to use any integer values for input for this PR.

Do you have any suggestion to range of random input data? =)

Reply:
@jinevening What do you think about [0, 127] range, which valid for any integer datatype?

Reply:
>  for the OneHot operation, the input data (indices) can be any.

For OneHot Op, I guess that the depth (range of indices) would be much smaller than the whole integer range. For example, for one-hot with depth = 1000, this random generator will make most of the output off-value.

How about using the range -100~100 (or -1000~1000)? It'll be possible to change the range if necessary. I think it would be better to take the range (min, max) as the arguments of this function.

Reply:
> I think it would be better to take the range (min, max) as the arguments of this function.

I think this is a good approach.

Reply:
> What do you think about [0, 127] range, which valid for any integer datatype?

This looks also good to me :)

Reply:
> > I think it would be better to take the range (min, max) as the arguments of this function.
> 
> I think this is a good approach.

I introduced 2 additional argument: min and max for generation helper. PTAL. =)